src/Console/Commands/CheckCommand.php

```diff
@@ -261,7 +261,18 @@ private function formatIssueSuggestionsForDisplay(Issue $issue): string
             $issue->misspelling->suggestions,
         );
 
-        return implode(', ', $suggestions);
+        $lastItem = array_pop($suggestions);
+
+        /**
+         * I have had to ignore this block.
+         * I have been unable to find a word that has just a single suggestion and will work cross-platform
```

Comment:
You can take these comments out, and justify this in the pull request descrition.
They don't need to stay in the code